### ISSUE TITLE
Place entities not resolving [JIRA CVRE-96]

### ISSUE THREAD

**Issue by cendarijiraimpert** [CENDARI/editorsnotes#97]
even when I verify and paste in the DBPedia link, some places don't seem to resolve (=save), and I have yet to get a map to display!

*Reporter*: Jennifer
*E-mail*: [mailto:edmondj@tcd.ie]

reported by TestCendariInria on 2015-02-20T13:22:23.445+0100

**Comment by cendarijiraimpert** [comment]
Can you tell me what DBPedia link you added?

Jean-DanielFekete on 2015-02-20T13:28:11.023+0100

**Comment by cendarijiraimpert** [comment]
I looked at your project and fixed the urls.
Actually, when you search on dbpedia, it returns pages called:
http://dbpedia.org/page/Constantinople
but the RDFa page is actually
http://dbpedia.org/resource/Constantinople

changing "/page/"  to "/resource/" fixes the problem and the map appears.
I have addressed that in the code, the change will be done automatically.

Jean-DanielFekete on 2015-02-20T17:04:00.017+0100
